python/hdfs_downloader.py

```diff
@@ -0,0 +1,245 @@
+"""
+Reusable parallel download-to-HDFS script
+"""
+import logging
+import os.path
+
+import hdfs
+import requests
+# from requests.packages.urllib3.exceptions import InsecurePlatformWarning
```

Comment:
I run the default version of python in altiscale (2.6), and this import has not removed the warning for me, so I commented them.
Uncommented now !